Community PR - Dune Dashboard (2a) (#5976)
w3f/polkadot-wiki: docs/general/dune-analytics/parachain-dashboards/litentry-dashboards.md

---
id: litentry-dashboards
title: Litentry Dashboards
sidebar_label: Litentry
description:
  Litentry is a decentralized identity aggregator that links user identities while ensuring privacy
  protection, thus enabling various social and economic innovations. It features a three-layer
  structure, comprising source data, address analysis, and identity aggregation layers, all secured
  with Trusted Execution Environment (TEE) technology to enhance security and privacy. Litentry's

>   applications include airdrop whitelisting, credit scoring, and cross-platform reputation building.

Reviewer: ```suggestion
description:
  Litentry is a decentralized identity aggregator that links user identities while ensuring privacy
  protection.
```

keywords: [polkadot, dashboard, dune, litentry, identity, LIT]
slug: ../litentry-dashboards
---

# Litentry Dashboards

## Overview

Litentry is a decentralized identity aggregator that links user identities while ensuring privacy
protection, thus enabling various social and economic innovations. It features a three-layer
structure, comprising source data, address analysis, and identity aggregation layers, all secured
with Trusted Execution Environment (TEE) technology to enhance security and privacy. Litentry's
applications include airdrop whitelisting, credit scoring, and cross-platform reputation building.

## Featured Dashboards on Dune

> Here you'll find a variety of dashboards that help visualize data from the Litentry parachain:

Reviewer: ```suggestion
Here you will find a variety of dashboards that help visualize data from the Litentry parachain:
```

- [Litentry](https://dune.com/substrate/litentry): A comprehensive analysis of Litentry, including:
  Token Sent To Ethereum, Staking, Asset Analysis.

## Key Tables

Data from the Litentry parachain is organized into several key tables:

- `litentry.balances`
- `litentry.blocks`
- `litentry.calls`
- `litentry.events`,
- `litentry.extrinsics`
- `litentry.transfers`

## Useful Queries

Currently, there are no specific useful queries provided. Please check back later as this section

> will be updated with materialized queries for Litentry.

Reviewer: ```suggestion
Currently, there are no specific queries provided. 
```
